src/gwt/src/org/rstudio/studio/client/common/filetypes/TextFileType.java

```diff
@@ -288,6 +288,7 @@ public String getScriptInterpreter()
       results.add(commands.goToLine());
       results.add(commands.expandSelection());
       results.add(commands.shrinkSelection());
+      results.add(commands.renameInFile());
```

Comment:
I think you're right -- this should be scoped to R document types for now; we can move it to more file types later if we extend the rename tool.